cabal-testsuite/cabal-testsuite.cabal

```diff
@@ -115,6 +115,6 @@ custom-setup
   -- and due to Custom complexity and ConstraintSetupCabalMaxVersion
   -- it has to be the latest release version plus
   -- you have to use the latest cabal-install release
-  setup-depends: Cabal == 3.8.*,
-                 Cabal-syntax == 3.8.*,
+  setup-depends: Cabal == 3.10.*,
```

Comment:
Now this is smart! I could not guess such a solution.

Reply:
Yeah, a priori I thought that this would stay as is was, and the cabal-install in the CI would stay 3.8 also. But I must say I have no explanation for this. It's just a hunch.